bagitobjecttransfer/caais/admin.py

```diff
@@ -86,11 +87,11 @@ class DispositionAuthorityInlineAdmin(admin.TabularInline):
 
 
 class SourceOfMaterialInlineAdmin(admin.StackedInline):
-    ''' Admin for editing source(s) of material inline
-    '''
+    """Admin for editing source(s) of material inline"""
+
+    class Media(PipelineFormMedia):
+        """JavaScript to load for inline form"""
 
-    class Media:
-        ''' JavaScript to load for inline form '''
         js = (
             "https://cdnjs.cloudflare.com/ajax/libs/jquery.mask/1.14.16/jquery.mask.min.js",
             "caais/js/phoneNumberMask.js",
```

Comment:
I'm still getting a 500 error from here - if you go to the "CAAIS Metadata" admin page (http://localhost:1337/admin/caais/metadata) you'll see an error.

The app logs say:

ValueError: Missing staticfiles manifest entry for 'caais/js/phoneNumberMask.js'

Reply:
Clarification - you have to go to the change page for one of the CAAIS Metadata entries.

Reply:
Ah I didn't see `caais/js/phoneNumberMask.js`, I thought that was part of the CDN JS library too. And it makes sense I didn't get a error, since I didn't test the change page.

I've made a separate JS package `caais_admin_js` so the change page should be working correctly now.

Resolved by  https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/221/commits/97e8fea12190524d00c24359325f8162dc7ef037